.NET/Microsoft.Recognizers.Definitions/English/TimeZoneDefinitions.cs

```diff
@@ -0,0 +1,286 @@
+		public static readonly Dictionary<string, int> abbr2Minute = new Dictionary<string, int>
+		{
+			{ "aft", 270 },
+			{ "akst", -540 },
+			{ "akdt", -540 },
+			{ "abst", 180 },
+			{ "ast", -240 },
+			{ "arbst", 240 },
```

Comment:
What about ambiguous cases?

Reply:
They are abbreviations for timezones

Reply:
I know they are abbreviations. But abbreviations are not unique.
How does the code handle ambiguous cases? I.e. two abbreviations with different offsets? Like CST.

Reply:
And where did this list come from?